### DISCUSSION TITLE
login issues when changinf the web-host

### DISCUSSION THREAD

**Discussion by saleemalharir1** [semaphoreui/semaphore#1856] [Q&A]
![image](https://github.com/ansible-semaphore/semaphore/assets/97859803/8a52d014-aa33-4e91-8656-1d040e725427)

so when i update the link in the snap web-host 
and then i add this code to my nginx 

location /semaphore/ {
        proxy_pass http://localhost:3000/;
        proxy_set_header Host $http_host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;
        proxy_buffering off;
        proxy_request_buffering off;
    }
    location /semaphore/api/ws {
        proxy_pass http://localhost:3000/api/ws;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "upgrade";
        proxy_set_header Origin "";
        
        i can't see the login althought i have a user for semaphore

**Comment by fiftin** [comment]
You need to provide `web_host` option in configuration file: https://docs.semui.co/administration-guide/configuration#public-url

**Reply by saleemalharir1** [reply]
I installed semaphore using snap so I don't know where the config is also I added it using 
sudo snap set semaphore web-host = ""
